Libs/MRML/Logic/vtkMRMLApplicationLogic.h

```diff
@@ -219,6 +221,11 @@ class VTK_MRML_LOGIC_EXPORT vtkMRMLApplicationLogic
   /// Uses current scene's URL property, so the URL must be up-to-date when calling this method.
   void SaveSceneScreenshot(vtkImageData* screenshot);
 
+  /// Invokes "PauseRenderEvent"
```

Comment:
It is an internal detail that the application logic implements pause/resume rendering by emitting an event. Here we should describe _what_ the method does (maybe include details such as when it should be used, etc.). We may describe _how_ it does it in the implementation (in the .cxx file), if it needs explanation.